src/main/java/com/cachinginapp/enterprise/CachingInController.java

```diff
@@ -70,14 +106,36 @@ public Cache createCache(@RequestBody Cache cache) {
         return newCache;
 
     }
-
-    @DeleteMapping("/cache/id/")
-    public ResponseEntity deleteCache(@PathVariable("id") String id){
+    /**
+     * removes a cache with given id
+     *
+     * Given the parameter id, remove the cache that corresponds to this unique ID.
+     *
+     * Returns one of the following status codes:
+     * 200: if found
+     * 400: if null
+     *
+     * @param id a JSON representation of a cache object.
+     * @return message confirming the cache was removed.
+     */
+    @DeleteMapping("/cache/{id}/")
+    public ResponseEntity deleteCache(@PathVariable("id") int id){
 
         return new ResponseEntity(HttpStatus.OK);
     }
 
-
+    /**
+     * searchesCaches with searchTerm value
+     *
+     * TODO: Describe the purpose of this method
+     *
+     * Returns one of the following status codes:
+     * 200: if found
+     * 400: if null
+     *
+     * @param searchTerm TODO: What is this parameter?
+     * @return TODO: What does this return?
+     */
```

Comment:
added Javadoc - **Needs clarification for how this will be implemented upon completion**